publications-2024.bib

```diff
@@ -1251,6 +1251,26 @@ @Article{2024:talukdar.swaminathan.ea:stress-induced
   publisher = {Springer}
 }
 
+@PhDThesis{2024:thiele:error-controlled,
+  author  = {Thiele, Jan Philipp},
+  title   = {Error-controlled space-time finite elements, algorithms and implementations for nonstationary problems},
+  school  = {Leibniz University Hannover},
+  year    = 2024,
+  doi     = {10.15488/17916},
+  url     = {https://www.repo.uni-hannover.de/handle/123456789/18051}
+}
+
+@Misc{2024:thiele:idealii,
+  author  = {Thiele, Jan Philipp},
+  title   = {ideal.{II}: a Galerkin Space-Time Extension to the Finite Element Library deal.{II}},
+  year    = 2024,
+  url     = {https://arxiv.org/abs/2408.08840},
+  doi     = {10.48550/arXiv.2408.08840},
+  publisher = {arXiv},
+  preprint = {https://arxiv.org/abs/2207.04764},
```

Comment:
```suggestion
```
This belongs to the article below :)

Reply:
Ye olde Copy & Paste.. Good you spotted this!